packages/create-plugin/README.md

```diff
@@ -116,4 +116,4 @@ For more information see our [documentation](https://grafana.com/developers/plug
 
 ## Contributing
 
-We are always grateful for contributions! See [CONTRIBUTING.md](../CONTRIBUTING.md) for more information.
+We are always grateful for contributions! See [CONTRIBUTING.md](../../CONTRIBUTING.md) for more information.
```

Comment:
I searched this file for `../` but this seems to be the only instance. I haven't searched other files for similar broken links - maybe the reviewer could do this? You probably have more context than me on how this got broken, and whether other files may also have broken in a similar way.